serving/frontend/src/Components/Forms/rules.jsx

```diff
@@ -1,10 +1,56 @@
 export const minimumPropertyValues = new Map([
-    ["cha", -5],
-    ["con", -5],
+    ["str", -5],
     ["dex", -5],
+    ["con", -5],
     ["int", -5],
-    ["str", -5],
     ["wis", -5],
+    ["cha", -5],
     ["ac", 0],
-    ["hp", 1]
+    ["hp", 1],
+    ["perception", 0],
+    ["fortitude", 0],
+    ["reflex", 0],
+    ["will", 0],
+    ["focus", 0],
+    ["num_immunities", 0],
+    ["land_speed", 0],
+    ["fly", 0],
+    ["climb", 0],
+    ["swim", 0],
+    ["spells_nr_lvl_1", 0],
+    ["spells_nr_lvl_2", 0],
+    ["spells_nr_lvl_3", 0],
+    ["spells_nr_lvl_4", 0],
+    ["spells_nr_lvl_5", 0],
+    ["spells_nr_lvl_6", 0],
+    ["spells_nr_lvl_7", 0],
+    ["spells_nr_lvl_8", 0],
+    ["spells_nr_lvl_9", 0],
+    ["melee_max_bonus", 0],
+    ["avg_melee_dmg", 0],
+    ["ranged_max_bonus", 0],
+    ["avg_ranged_dmg", 0],
+    ["acid_resistance", 0],
+    ["all-damage_resistance", 0],
+    ["bludgeoning_resistance", 0],
+    ["cold_resistance", 0],
+    ["electricity_resistance", 0],
+    ["fire_resistance", 0],
+    ["mental_resistance", 0],
+    ["physical_resistance", 0],
+    ["piercing_resistance", 0],
+    ["poison_resistance", 0],
+    ["slashing_resistance", 0],
```

Comment:
![image](https://github.com/Paulina100/ML-for-TTRPG-game-design/assets/92922548/0eb8b3a7-6856-4c05-bfcf-e8bf43ef030d)

Takie coś mi się wyświetla chodź nie wiem czemu

Reply:
O chyba jak są nazwy zmiennych, które mają w sobie "-" to się wyświetla jako undefined